net/core/skbuff.c

```diff
@@ -260,8 +260,8 @@ do {									\
 
 	pools = this_cpu_ptr(pg_mpool);
 
-	for (o = (cn == 1) ? 0 : (cn == 2) ? 1 : (cn <= 4) ? 2 : PG_LISTS_N;
-	     o < PG_LISTS_N; ++o)
+	for (o = (cn == 1) ? 0 : (cn == 2) ? 1 : (cn <= 4) ? 2 : (cn <= 8) ? 3 :
+	    (cn <= 16) ? 4 : PG_LISTS_N; o < PG_LISTS_N; ++o)
```

Comment:
The loop initialization becomes too ugly
```suggestion
        o = (cn == 1) ? 0                                                                                           
            : (cn == 2) ? 1 
              : (cn <= 4) ? 2                                                                                       
                : (cn <= 8) ? 3                                                                                     
                  : (cn <= 16) ? 4 : PG_LISTS_N                                                                     
        for ( ; o < PG_LISTS_N; ++o) {               
```